spec/features/get_a_teacher_relocation_payment/teacher_route_completing_the_form_spec.rb

```diff
@@ -41,31 +45,33 @@
       it "submits an application" do
         and_i_complete_the_nationality_step_with(option: "Australian")
         and_i_complete_the_passport_number_step_with(options: "123456789")
-        and_i_complete_the_employment_details_step
+        and_i_complete_the_current_school_step(school)
+        and_i_complete_the_headteacher_step
         and_i_complete_the_personal_details_step
         and_i_complete_the_postcode_step
         and_i_complete_the_email_address_step
         and_i_dont_provide_my_mobile_number
         and_i_provide_my_personal_bank_details
         and_i_complete_the_payroll_gender_step
-        then_the_check_your_answers_part_page_shows_my_answers
+        then_the_check_your_answers_part_page_shows_my_answers(school)
         and_i_submit_the_application
         then_the_application_is_submitted_successfully
       end
     end
 
     context "without postcode search" do
-      it "submits an application" do
+      it "submits an application", js: true do
```

Comment:
Figured we want to hit the js version of the current school form in one of the IRP tests to make sure that's covered by a spec